symbols/overlay29.yml

```diff
@@ -8541,6 +8555,15 @@ overlay29:
         
         r0: mission destination info pointer
         return: bool
+    - name: MissionIsActiveOnFloor
```

Comment:
Since this returns the mission pointer itself rather than a bool, I think we should use a different naming convention, e.g. `GetMissionIfActive`